cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/WorkflowHttpHandler.java

```diff
@@ -207,15 +209,16 @@ public void getWorkflowToken(HttpRequest request, HttpResponder responder,
                                    workflowTokenNodeDetailType));
   }
 
-  private WorkflowToken getWorkflowToken(String namespaceId, String appName, String workflow,
+  private WorkflowToken getWorkflowToken(String namespace, String appName, String workflow,
```

Comment:
@chtyim for this one too - should we move the whole logic to store.getWorkflowToken(namespaceId , appName, workflowId, runId) instead? which also fetches latest?